File: app/src/main/java/com/example/practicals/java_practicals/Operators.java
package com.example.practicals.java_practicals;

import java.io.*;
import java.util.Scanner;
public class Operators {
    public static void main(String[] args) {
        int a = 10, b = 2;
        System.out.print("1.arithmetic"+"\n"+"2.relational"+"\n"+"3.bitwise"+"\n"+"4.logical"+"\n"+"5.assignment"+"\n"+"6.misc"+"\n");
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();

        switch (str) {
            case "1" :  System.out.println("a + b = "+ (a + b));
                System.out.println("a - b = " + (a - b));
                System.out.println("a * b = " + (a * b));
                System.out.println("a / b = " + (a / b));
                System.out.println("a % b = " + (a % b));

            case "2" :  System.out.println(a == b);
                System.out.println(a != b);
                System.out.println(a > b);
                System.out.println(a < b);
                System.out.println(a >= b);
                System.out.println(a <= b);

            case "3" :  System.out.println("a&b = " + (a & b));
                System.out.println("a|b = " + (a | b));
                System.out.println("a^b = " + (a ^ b));
                System.out.println("~a = " + ~a);

            case "4" :
                System.out.println((a > 5) && (b > 1));
                System.out.println((a > 5) || (b > 3));
                System.out.println(!(a == b));

            case "5" :
                int var = a;
                System.out.println("var using =: " + var);
                var += a;
                System.out.println("var using +=: " + var);
                var *= a;
                System.out.println("var using *=: " + var);

            case "6" :
                System.out.println( "grater number is: " +  ((a > b) ? a : b ) );
        }
    }
}

File: app/src/main/java/com/example/practicals/kotlin_practicals/Operators.kt
package com.example.practicals.kotlin_practicals;

public class Operators {
}
fun main() {
    var a=10;
    var b=2;

    println("Arithmetic")
    println("a+b : "+(a+b))
    println("a-b : "+(a-b))
    println("a*b : "+(a*b))
    println("a/b : "+(a/b))
    println("a%b : "+(a%b))
    println()

    println("Relational")
    println("a>b : "+(a>b))
    println("a<b : "+(a<b))
    println("a>=b : "+(a>=b))
    println("a<=b : "+(a<=b))
    println("a==b : "+(a==b))
    println("a!=b : "+(!(a.equals(b))))
    println()

    println("Unary")
    println("+a :"+ ++a)
    println("-b :"+ b--)
    println("++a :"+ ++a)
    println("--b :"+ --b)
    println()

    println("Logical")
    println((a > 5) && (b > 1))
    println((a > 5) || (b > 3))
    println(!(a == b))
    println()

    println("Assignment")
    a=10
    b=2
    a+=b
    println("a+=b : "+a)
    a-=b
    println("a-=b : "+a)
    a*=b
    println("a*=b : "+a)
    a/=b
    println("a/=b :"+a)
    a%=b
    println("a%=b :"+a)
    println()

    println("Bitwise")
    println("a.and(b): "+a.and(b))
    println("a.or(b): "+a.or(b))
    println("a.xor(b): "+a.xor(b))
    println("a.inv(): "+a.inv())
    println("a.shl(b): "+a.shl(b))
    println("a.shr(b): "+a.shr(b))
}
